### PR TITLE
added set up to README

### PR DESCRIPTION
**Pull request by shukii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ The tool will be made available to the public.
 Please help contribute.
 
 Thank you
+
+# Getting set up
+
+* Run `npm install`
+* To start the dev version, which will serve the application locally, run `npm run start`. It will serve it on localhost:4200, and hot reload as you change the code
```
Comment (stevebrowndotco) [inline README.md]: `ionic serve`, i dont think `npm run start` works

Comment (shukii) [reply README.md]: Have changed npm start script to use Ionic Serve, user will need to install it though

**Comment by stevebrowndotco** [comment]
@shukii Hey thanks so far ;) Question, where did the package json changes come from? Lots of packages installed, was this intentional?

**Comment by stevebrowndotco** [comment]
Oh i see, the package was just re-ordered

**Comment by shukii** [comment]
> @shukii Hey thanks so far ;) Question, where did the package json changes come from? Lots of packages installed, was this intentional?

Reordered -  probably my ide settings

**Pull request merged**